lib/envied/coercer.rb

```diff
@@ -16,13 +19,14 @@ def coerce(string, type)
     unless supported_type?(type)
       raise ArgumentError, "#{type.inspect} is not supported type"
     end
-    coerce_method_for(type.to_sym)[string]
+    # coerce_method_for(type.to_sym)[string]
+    coercer.public_send("to_#{type.downcase}".to_sym, string)
   end
 
-  def coerce_method_for(type)
-    return nil unless supported_type?(type)
-    coercer.method("to_#{type.downcase}")
-  end
+  # def coerce_method_for(type)
+  #   return nil unless supported_type?(type)
+  #   coercer.method("to_#{type.downcase}")
+  # end
```

Comment:
Q: Is this code going away? (Can it already be deleted as opposed to be commented-out?)

Reply:
Yes. It will be deleted. Instead of a hard delete, its easier for me to uncomment as I experiment but I'm much more sure now this and other things will be removed. I'm copying over all the test cases from coercible that apply for the types we support so I can be reassured that this and other code can be deleted permanently.